skyvern/forge/sdk/services/credential/credential_vault_service.py

```
@@ -26,6 +26,12 @@ async def create_credential(self, organization_id: str, data: CreateCredentialRe
     async def delete_credential(self, credential: Credential) -> None:
         """Delete a credential from the vault and database."""
 
+    async def post_delete_credential_item(self, item_id: str) -> None:
+        """
+        Optional hook for scheduling background cleanup tasks after credential deletion.
+        Default implementation does nothing. Override in subclasses as needed.
+        """
```

Ruff (0.14.0)

`CredentialVaultService.post_delete_credential_item` is an empty method in an abstract base class, but has no abstract decorator

(B027)